app/javascript/dashboard/components/buttons/ResolveAction.vue

```diff
@@ -143,12 +128,10 @@ export default {
     },
   },
   mounted() {
-    bus.$on(CMD_SNOOZE_CONVERSATION, this.onCmdSnoozeConversation);
```

Comment:
I moved the `CMD_SNOOZE_CONVERSATION` bus event from the `ResolveAction.vue` to the `ChatList.vue` so that the same event could be used from the chat list and chat view.